src/player.cpp

```diff
@@ -74,6 +74,8 @@ Player::Player(const char *name, IItemDefManager *idef):
 
 	hud_hotbar_itemcount = HUD_HOTBAR_ITEMCOUNT_DEFAULT;
 
+	m_fov = 0.0;
```

Comment:
```suggestion
	m_fov = 0.0f;
```

Reply:
As per C++11 standards, `m_fov` should actually be defined inline, as the value is a constant.

Reply:
Fixed, thanks